forest/rdt.py

```diff
@@ -342,8 +342,13 @@ def load_tail_points(path):
     @staticmethod
     def load_centre_points(path):
         """Holds a centre point, future point and future movement line"""
-        with open(path) as stream:
-            rdt = json.load(stream)
+        if os.path.splitext(path)[1] == '.json':
+            with open(path) as stream:
+                rdt = json.load(stream)
+        elif os.path.splitext(path)[1] == '.nc':
+            getRDT(path, 0, 'Point')
+        else:
+            return 'Can\'t read this file type'
 
```

Comment:
@claretandy to me it looks like `load_centre_points` is mostly parsing the contents of the JSON text files.

Would it be better to split them?

```python
def load_centre_points(...):
    if ... == '.nc':
        return self.load_centre_points_netcdf(path)
    else:
        return self.load_centre_points_json(path)
```